content/docs/reference-react.md

```diff
@@ -176,7 +176,11 @@ Returns the total number of components in `children`, equal to the number of tim
 React.Children.only(children)
 ```
 
-Returns the only child in `children`. Throws otherwise.
+Verifies `children` has only one child and the child is a React element, then returns it. Throws otherwise.
+
+> Note:
+>
+>`React.Children.only()` does not accept as argument a return value from [`React.Children.map()`](#reactchildrenmap) because the return of the latest is not a React element.
```

Comment:
What do you think about

> Verifies that `children` has only one child (a React element) and returns it. Otherwise this method throws.

>> Note:
>>
>> `React.Children.only()` does not accept the return value of [`React.Children.map()`](#reactchildrenmap) because it is an array rather than a React element.

Reply:
Looks great. I will make the change right now.